docs/en/04_Changelogs/4.0.0.md

```diff
@@ -415,8 +419,14 @@ require manual intervention, please see the below upgrading notes.
 Templates are now much more strict about their locations. You can no longer put a template in an arbitrary
 folder and have it be found. Case is now also checked on case-sensitive filesystems.
 
-Either include the folder in the template name (`renderWith('Field.ss')` => `renderWith('forms/Field.ss')`), move the 
-template into the correct directory, or both.
+Either include the folder in the template name (`renderWith('MyEmail.ss')` => `renderWith('emails/MyEmail.ss')`),
+move the template into the correct directory, or both.
+
+When using `<% include %>` template tag you should continue to leave out the `Includes` folder. 
```

Comment:
This advice is only accurate until we implement the suggestions in https://github.com/silverstripe/silverstripe-framework/issues/5952#issuecomment-244881303 (where `<% include Includes/Sidebar %>` will become the new and consistent default)